### ISSUE TITLE
Lightvote System

### ISSUE THREAD

**Issue by Fyrsta** [TranxCraft/iTranxCraft#39]
For those that are not familiar with [Lightvote](http://dev.bukkit.org/bukkit-plugins/lightvote/Lightvote), it's a plugin that allows players to vote if they want the world to be day or night. It's a nice plugin to have however it would be better if it could be integrated into this plugin.

**Comment by Wild1145** [comment]
Personally dont agree with the time being able to be changed... Weather yes, time, no. Shouldn't be needed for players...

**Comment by Fyrsta** [comment]
@Wild1145 It's a plugin that I like. It allows the world to be whatever the majority of the players want it to be. We can integrate a cooldown in the system is needs be where if it was voted for day/night the time before, you won't be able to start another vote. This'll stop a group of players keeping the world in the same time schedule. I personally don't see the point of weather voting as it has very little effect on players. The only time when it has an effect is when there's lightning and it hits a creeper. That's about it.

**Comment by Wild1145** [comment]
@WickedGamingUK I know it has its uses, but for a survival server, to me it just defeats the object. If you want to go out monster hunting or something and everyone votes against you, and they do it every time, you can never get the stuff you need without buying it from a server shop. Its a personal preference thing for me, where the time should be controlled by the server staff... Weather can cause lag and in some biomes I believe allow for mob spawns and such, but its such a significantly more minor thing to have, it wouldn't be a big deal to have players vote to change it in my opinion.

**Comment by Fyrsta** [comment]
@Wild1145 That is true, hence the suggestion of a cooldown so it's impossible to not have at least one day or night cycle. Weather voting is possible to do, and if it does cause lag in some biomes then it may as well be implemented.
